cli/src/main/resources/rules/no_gpl_declared.kts

```diff
@@ -2,8 +2,13 @@
 //
 // $ ort evaluate -i scanner/src/funTest/assets/file-counter-expected-output-for-analyzer-result.yml --rules-resource /rules/no_gpl_declared.kts
 
-ortResult.analyzer?.result?.packages?.all { curatedPkg ->
-    curatedPkg.pkg.declaredLicenses.none { license ->
+val pkgWithGpl = ortResult.analyzer?.result?.packages?.filter { curatedPkg ->
+    curatedPkg.pkg.declaredLicenses.any { license ->
         license == "GPL" || license.startsWith("GPL ") || license.startsWith("GPL-")
```

Comment:
A possible improvement for later would be to use `LicenseMapping.map(license)` and check if this maps to any of the GPL licenses from the `SpdxLicense` enum. This would also catch some weird declared licenses.

Reply:
Basically a nice idea! However, the number of enums from `SpdxLicense` to check for would be quite large. I doubt that it would improve readability in the end.